<ui version="4.0" >
 <class>UpdatesDialog</class>
 <widget class="QDialog" name="UpdatesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Update</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="pixmapLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QLabel" name="descriptionLabel" >
     <property name="minimumSize" >
      <size>
       <width>400</width>
       <height>96</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QWidget" native="1" name="releaseNotesGroup" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&lt;b>Release Notes:&lt;/b></string>
          </property>
          <property name="textFormat" >
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDUPDATERVIEW" native="1" name="releaseNotesView" >
          <property name="minimumSize" >
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" native="1" name="packageSwitchBar" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="packageUpdateCheckBox" >
          <property name="text" >
           <string>Update the current package</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>296</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="previousPackageButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="iconSize" >
           <size>
            <width>8</width>
            <height>8</height>
           </size>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
          <property name="arrowType" >
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentPackageLabel" >
          <property name="text" >
           <string>N</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalPackageLabel" >
          <property name="text" >
           <string>M</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="nextPackageButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="iconSize" >
           <size>
            <width>8</width>
            <height>8</height>
           </size>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
          <property name="arrowType" >
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDUPDATERVIEW</class>
   <extends>QWidget</extends>
   <header>qwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
